File: JavaScript/code_wars/unique_in_order.js
var uniqueInOrder = function (iterable) {
  let unique = [];

  if (iterable.length === 0) return unique;

  for (const val of iterable) {
    if (unique.length > 0 && unique[unique.length - 1] === val) continue;
    unique.push(val);
  }

  return unique;
};

File: Python/code_wars/unique_in_order.py
def unique_in_order(sequence):
    unique = []
    if len(sequence) == 0:
        return []

    for val in sequence:
        print(len(unique))
        if len(unique) > 0 and val == unique[len(unique) - 1]:
            continue
        unique.append(val)

    return unique
